docs/development/Run-business-tests-local.md

```diff
@@ -128,6 +128,14 @@ kubectl get svc -n business-tests -o go-template='{{range .items}}{{ $save := .
 This will return all NodePorts which are available in business-tests namespace where you can pick the ports to use in your environment variables.
 Now you are able to run it in IDE either as normal "Run" mode or in "Debug" mode where you can debug the business-tests by setting debugging points.
 
+Example of mapping to environment variables needed for the business tests:
+
+```shell
+business-tests/plato-controlplane - data: 30955(8081)    -> PLATO_DATA_MANAGEMENT_URL=http://localhost:30955/data;
+business-tests/sokrates-controlplane - data: 30538(8081) -> SOKRATES_DATA_MANAGEMENT_URL=http://localhost:30538/data;
+business-tests/backend - backend: 30556(8081)            -> SOKRATES_BACKEND_SERVICE_BACKEND_API_URL= http://localhost:30556
```

Comment:
another idea could be to prepare a template `.env` file, where people need only fill in some ports, and can then use it in the intelliJ run config